src/utils.js

```diff
@@ -166,7 +177,7 @@ export function isFunction(target) {
  * @return {boolean}
  * @private
  */
-export function isString(target) {
+export function isString(target: *): boolean {
```

Comment:
The existential type! It says "let flow infer it"

https://flow.org/en/docs/types/utilities/#toc-the-existential-type

But basically, any value is permissible here.